keras_nlp/models/gpt2/gpt2_backbone.py

```diff
@@ -144,7 +145,10 @@ def __init__(
                 kernel_initializer=_gpt_2_kernel_initializer(stddev=0.02),
                 normalize_first=True,
                 name=f"transformer_layer_{i}",
-            )(x, decoder_padding_mask=padding_mask)
```

Comment:
is there a reason for this change? actually found it more readable as before

Reply:
Black will go with the former version since the line is short enough